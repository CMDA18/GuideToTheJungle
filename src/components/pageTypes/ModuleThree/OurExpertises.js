// @flow
import React, { Component } from 'react'
import Helmet from 'react-helmet'
// import styled from 'styled-components'
import Grid from '../../Grid'
import PageTitle from '../../PageTitle'
import Intro from '../../Intro'
import { ourExpertises } from '../../../constants/content/index'

type Props = {
  test: string
}

class OurExpertises extends Component<Props> {
  render () {
    return (
      <div className='ourExpertises' data-test={this.props.test}>
        <Helmet
          title='ourExpertises'
        />
        <Grid page='jungleMinds'>
          <header>
            <PageTitle {...ourExpertises}/>
            <Intro {...ourExpertises}/>
          </header>
        </Grid>
      </div>
    )
  };
}

export default OurExpertises
