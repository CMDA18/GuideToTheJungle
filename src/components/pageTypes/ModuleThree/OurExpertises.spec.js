import React from 'react'
import { shallow } from 'enzyme'
import OurExpertises from './OurExpertises'

it('renders without crashing', () => {
  expect(shallow(<OurExpertises />)).toMatchSnapshot()
})
