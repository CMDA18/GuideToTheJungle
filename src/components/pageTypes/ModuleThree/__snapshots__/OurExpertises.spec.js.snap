// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OurExpertises />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          title="ourExpertises"
        />,
        <Grid
          page="jungleMinds"
        >
          <header>
            <PageTitle />
            <Intro />
          </header>
        </Grid>,
      ],
      "className": "ourExpertises",
      "data-test": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "defer": true,
          "encodeSpecialCharacters": true,
          "title": "ourExpertises",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <header>
            <PageTitle />
            <Intro />
          </header>,
          "page": "jungleMinds",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <PageTitle />,
              <Intro />,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "header",
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defer={true}
            encodeSpecialCharacters={true}
            title="ourExpertises"
          />,
          <Grid
            page="jungleMinds"
          >
            <header>
              <PageTitle />
              <Intro />
            </header>
          </Grid>,
        ],
        "className": "ourExpertises",
        "data-test": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "defer": true,
            "encodeSpecialCharacters": true,
            "title": "ourExpertises",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <header>
              <PageTitle />
              <Intro />
            </header>,
            "page": "jungleMinds",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <PageTitle />,
                <Intro />,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "header",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
